tests/conf/ssl4eo_l_downstream_cdl.yaml

```diff
@@ -0,0 +1,20 @@
+module:
+  _target_: torchgeo.trainers.SemanticSegmentationTask
+  loss: "ce"
+  model: "unet"
+  backbone: "resnet18"
+  weights: null
+  learning_rate: 1e-3
+  learning_rate_schedule_patience: 6
+  in_channels: 7
+  num_classes: 17
+  num_filters: 1
+  ignore_index: 0
+
+datamodule:
+  _target_: torchgeo.datamodules.SSL4EOLDownstreamDataModule
+  root: "tests/data/ssl4eo_downstream_landsat"
+  input_sensor: "tm_toa"
+  mask_product: "cdl"
+  batch_size: 2
+  num_workers: 0
```

Comment:
Can you add an EOL char before this EOF char? That will remove the red circle

Reply:
In the files changed tab, not in the file. Also, you'll see it in the git diff, it will say "no EOL character" or something like that.